.github/workflows/pull_request.yml

```diff
@@ -34,16 +34,15 @@ jobs:
     steps:
       # :-( https://github.community/t/support-for-yaml-anchors/16128
       - {uses: actions/checkout@v2, with: {fetch-depth: 0}}
-      - uses: actions/setup-python@v2
+      - run: python --version
 
       # Package index is "out of date by design"
       # https://github.com/actions/virtual-environments/issues/1757
       - name: apt-get -y update
         run: sudo apt-get -y update
 
       - name: get python libs
-        # FIXME: apt-get succeeds but 'import numpy' still fails!?
-        run: sudo apt-get -y install python3-numpy python3-scipy pylint
+        run: sudo apt-get -y install $(cat python-deps.txt)
```

Comment:
Same here, let's merge this before even looking at the rest and get them out of the way.